en/02_Developer_Guides/08_Performance/05_Resource_Usage.md

```diff
@@ -1,33 +0,0 @@
----
```

Comment:
By itself this is a pretty weak page. I've moved its contents to "Other Optimisations"